### PR TITLE
excluded whitelisted master branch form optaplanner PRs

### PR DESCRIPTION
**Pull request by mbiarnes**
**Thank you for submitting this pull request**

**JIRA**: _(please edit the JIRA link if it exists)_ 

[link](https://examle.com)

**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_

* referenced pull request 1
* referenced pull request 2
* referenced pull request 2
etc. 

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

@rsynek @Ginxo optaplanner PR has not any more "master" as white listed branch

### CODE REVIEW

**Review by rsynek — APPROVED** [review]
Thanks!

File: job-dsls/jobs/pr_jobs.groovy [hunk 1]
```diff
@@ -206,12 +206,20 @@ for (repoConfig in REPO_CONFIGS) {
...
-                    ghprbBranch {
-                        branch("7.x")
+                } else {
+                    whiteListTargetBranches {
+                        ghprbBranch {
+                            branch("${repoBranch}")
+                        }
+                        ghprbBranch {
+                            branch("7.x")
+                        }
```
Comment (Ginxo) [inline job-dsls/jobs/pr_jobs.groovy:222]: I would suggest to replace your changes by
```
whiteListTargetBranches {
    if ( repo != "optaplanner" ) {
        ghprbBranch {
            branch("${repoBranch}")
        }
    }
    ghprbBranch {
        branch("7.x")
    }
}
```
wdyt @mbiarnes ?

**Review by Ginxo — CHANGES_REQUESTED** [review]
please check

File: job-dsls/jobs/pr_jobs.groovy (hunk 1 above)
Comment (mbiarnes) [reply job-dsls/jobs/pr_jobs.groovy:222]: @Ginxo are we not interested in having master and 7.x branch in optaweb-employee-rostering and optaweb-vehicle-routing?

Comment (Ginxo) [reply job-dsls/jobs/pr_jobs.groovy:222]: @mbiarnes I can't tell about business here, just saying your PR proposal
```
if ( repo == "optaplanner" ) {
    whiteListTargetBranches {
        ghprbBranch {
            branch("7.x")
        }
    }
} else {
    whiteListTargetBranches {
        ghprbBranch {
            branch("${repoBranch}")
        }
        ghprbBranch {
            branch("7.x")
        }
    }
}
```
can be replace by this which is simpler 
```
whiteListTargetBranches {
    if ( repo != "optaplanner" ) {
        ghprbBranch {
            branch("${repoBranch}")
        }
    }
    ghprbBranch {
        branch("7.x")
    }
}
```

Comment (mbiarnes) [reply job-dsls/jobs/pr_jobs.groovy]: DONE

Comment (mbiarnes) [reply job-dsls/jobs/pr_jobs.groovy]: @Ginxo mrge when you're OK with it

**Pull request merged**